test/salt/pillar/default.sls

```diff
@@ -1,3 +1,6 @@
+# -*- coding: utf-8 -*-
```

Comment:
are those warnings needed for all YML files? for all YML files not names .yml or .yaml?

Reply:
Do you mean supplying the file to `yamllint`?  If so, then that was covered in this comment: https://github.com/saltstack-formulas/apt-formula/pull/49#discussion_r311670959.

Reply:
rephrased: "do yamllint needs this header in all YAML files?"
But I think not, so maybe it's `ssf-formula` which adds it.
If you think they are useful, I'm ok with it.

Reply:
OK, I misunderstood.  I only brought that in as a consistent header used in all of our existing YAML files in the `template-formula`.  It's not needed for `yamllint`.  If this needs to be fixed in any way, we could arrange that in the `template-formula` first and then get that propagated across to all other formulas.

Reply:
The headers are for guys that use something like ```vim```, that way vim will use the right syntax highlighting without worrying which extension to use, and setting the correct format

Reply:
Yes, I know, I'm myself a vim user :) But I use the `salt-vim` and maybe it automatically provides this helper. I just don't normally put this header (the vim one, not the encoding one which is fine) because vim, then emacs, then sublime, then... it's a never-ending need ;)